src/stdweb.rs

```diff
@@ -10,7 +10,8 @@ use crate::Error;
 extern crate std;
 use std::thread_local;
 
-use stdweb::js;
+#[macro_use]
+extern crate stdweb;
```

Comment:
Is there a reasons for this change?

Reply:
Whoops, nope. It was a rebase error. Removed